client/scss/grids/_grid_position.scss

```diff
@@ -0,0 +1,25 @@
+/* alignment modifiers per row */
+.grid--top {
+  align-items: flex-start;
+}
+
+.grid--bottom {
+  align-items: flex-end;
+}
+
+.grid--center {
+  align-items: center;
+}
+
+/* alignment modifiers per cell */
+.grid__cell--top {
+  align-self: flex-start;
+}
+
+.grid__cell--bottom {
+  align-self: flex-end;
+}
+
+.grid__cell--center {
+  align-self: center;
+}
```

Comment:
Can you add a rule to get your editor to ensure there's a final newline at the end of files?